Mathlib/Data/Set/Subsingleton.lean

```diff
@@ -276,19 +276,10 @@ theorem nontrivial_of_nontrivial (hs : s.Nontrivial) : Nontrivial α :=
   ⟨⟨x, y, hxy⟩⟩
 #align set.nontrivial_of_nontrivial Set.nontrivial_of_nontrivial
 
--- Porting note: simp_rw broken here
--- Perhaps review after https://github.com/leanprover/lean4/issues/1937?
 /-- `s`, coerced to a type, is a nontrivial type if and only if `s` is a nontrivial set. -/
 @[simp, norm_cast]
 theorem nontrivial_coe_sort {s : Set α} : Nontrivial s ↔ s.Nontrivial := by
-  -- simp_rw [← nontrivial_univ_iff, Set.Nontrivial, mem_univ, exists_true_left, SetCoe.exists,
-  --   Subtype.mk_eq_mk]
```

Comment:
This old `simp_rw` does still not work, but `exists_true_left` seems to be pretty unrelated to the goal state. Since it's a short `simp` statement, I thought, maybe it just ought to be replaced.